### PR TITLE
Feature/smoke tests

### PR DESCRIPTION
**Pull request by andyhd**

### CODE REVIEW

**Comment by tyom** [comment]
Should we add version number to JSON?

**Comment by andyhd** [comment]
Yep, good idea

**Comment by andyhd** [comment]
Most recent tag is now included in the response, but the code to generate that is in cla_common (https://github.com/ministryofjustice/cla_common/commit/e9df5a7dae2ce9af615041d1eb02227e14345610)

**Comment by s-block** [comment]
Looks good to me

File: cla_public/apps/checker/tests/smoketests.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        with self.app.test_client() as client:
+            zendesk.tickets()
+
+    def test_can_access_laalaa(self):
+        "search for legal advisers on LAALAA"
+        with self.app.test_client() as client:
+            laalaa.find('sw1a1aa')
+
+    @unittest.skip('not merged yet')
+    def test_can_access_mailgun(self):
```
Comment (tyom) [inline cla_public/apps/checker/tests/smoketests.py]: Not using Mailgun anymore.

**Pull request merged**